spec/models/comment_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] You have done well adding tests for the User, Post, and Comment models, but all models should be tested. Kindly add the missing tests for the Like model also.

Reply:
Added